buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -444,45 +439,46 @@ class BeamModulePlugin implements Plugin<Project> {
         commons_lang3                               : "org.apache.commons:commons-lang3:3.9",
         commons_math3                               : "org.apache.commons:commons-math3:3.6.1",
         error_prone_annotations                     : "com.google.errorprone:error_prone_annotations:2.0.15",
-        gax                                         : "com.google.api:gax:$gax_version",
-        gax_grpc                                    : "com.google.api:gax-grpc:$gax_version",
+        gax                                         : "com.google.api:gax",
+        gax_grpc                                    : "com.google.api:gax-grpc",
         google_api_client                           : "com.google.api-client:google-api-client:$google_clients_version",
         google_api_client_jackson2                  : "com.google.api-client:google-api-client-jackson2:$google_clients_version",
         google_api_client_java6                     : "com.google.api-client:google-api-client-java6:$google_clients_version",
-        google_api_common                           : "com.google.api:api-common:1.8.1",
+        google_api_common                           : "com.google.api:api-common",
```

Comment:
A question for my learning. Previously this file had specific dependency versions. For any released Beam version we could check the file in the release branch and get a list of dependencies and their versions.

After this change, how can we do the same thing? Would it happen through a generated BOM file in the release branch?

Reply:
OK, this is good.
- For easy access could we link to that dashboard from the place we are including the bom in BeamModulePlugin.groovy

One clarification question. Beam will depend on that exact list of dependencies shown in that dashboard. Is that correct?

Reply:
Added the link and comments ` // google_cloud_platform_libraries_bom sets version`. They may look too verbose, but they'll help future question "_where is the version of this artifact?_", especially when we start using other BOMs than GCP Libraries BOM in future.


> Beam will depend on that exact list of dependencies shown in that dashboard.

Not exact list. Beam will pick up the version from the BOM for Maven artifacts without a version. 
For example,
- Beam will depend on the BOM's gax version "1.54.0", because Beam's dependency `com.google.api:gax` does not specify a version.
- Beam **will not** depend on the BOM's Guava version "28.2-android", because Beam explicitly specify Guava's version "25.1-jre" as in `com.google.guava:guava:$guava_version`.
- Beam **will not** depend on the BOM's `com.google.http-client:google-http-client-apache-v2:1.34.2`, because Beam does not depend on `com.google.http-client:google-http-client-apache-v2`.